doc/schema/relationship-types/belongs-to.md

```diff
@@ -1,10 +1,10 @@
-# belongsTo( [schemaName, options] )
+# belongsTo(schemaName?, options?)
 
 `belongsTo()` creates a relationship from the current _source_ schema to the _target_ schema. The following makes each account
 belong to a sales person:
 
 ```ts
-const Account: PartialSchema = {
+const SalesPerson: PartialSchema = {
```

Comment:
i think our documentation should show `const SalesPerson = { ... } satisfies PartialSchema` because with these annotations we will lose the TS support on frontend